sky/tools/create_macos_framework.py

```diff
@@ -87,7 +87,7 @@ def main():
       'lipo', arm64_dylib, x64_dylib, '-create', '-output', fat_framework_binary
   ])
   # Make the framework readable and executable: u=rwx,go=rx.
-  subprocess.check_call(['chmod', '755', fat_framework_binary])
+  subprocess.check_call(['chmod', '755', fat_framework])
```

Comment:
I wonder if we should rename this one something like `fat_framework_dir` for clarity?

Reply:
Alternatively, `fat_framework_folder` which has the whole alliteration thing going for it.

Reply:
maybe `fat_framework_bundle`? The `fat_framework_dir` could be the directory the framework is in?

Reply:
Nice -- I like it; definitely less ambiguous.